src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -6901,6 +6915,7 @@ uint32 Unit::SpellHealingBonusDone(Unit* victim, SpellInfo const* spellProto, ui
         return healamount;
 
     int32 DoneTotal = 0;
+    //@TODO : Rewrite this function and add Versatility
```

Comment:
todo? what about few lines below that?

Reply:
I guess it's obsolete now. Since i added calculation to SpellHealingPctDone.